frontend/src/components/SchemaCreate.vue

```diff
@@ -1,212 +1,22 @@
-              return { value: type, text: ATTR_TYPES_NAMES[type] };
-            })
-          "
-          :args="{ id: `attrType${rowIndex}` }"
-        />
-      </div>
-
-      <div v-if="this.attributeDefinitions[rowIndex].type == 'FK'" class="col">
-        <Select
-          label="Bind to schema"
-          v-model="this.attributeDefinitions[rowIndex].bind_to_schema"
-          :options="
-            this.schemasToBind.map((schema) => {
-              return { value: schema.id, text: schema.name };
-            })
-          "
-          :args="{ id: `boundFK${rowIndex}` }"
-        />
-      </div>
-
-      <div class="col-lg-3">
-        <Textarea
-          label="Description (optional)"
-          v-model="this.attributeDefinitions[rowIndex].description"
-          :args="{ id: `description${rowIndex}` }"
-        />
-      </div>
-    </div>
-
-    <div class="row">
-      <div class="form">
-        <div class="form-check form-check-inline">
-          <Checkbox
-            label="Key"
-            v-model="this.attributeDefinitions[rowIndex].key"
-            :args="{ id: `key${rowIndex}` }"
-          />
-        </div>
-        <div class="form-check form-check-inline">
-          <Checkbox
-            label="Unique"
-            v-model="this.attributeDefinitions[rowIndex].unique"
-            :args="{ id: `unique${rowIndex}` }"
-          />
-        </div>
-        <div class="form-check form-check-inline">
-          <Checkbox
-            label="Required"
-            v-model="this.attributeDefinitions[rowIndex].required"
-            :args="{ id: `required${rowIndex}` }"
-          />
-        </div>
-        <div class="form-check form-check-inline">
-          <Checkbox
-            label="List"
-            v-model="this.attributeDefinitions[rowIndex].list"
-            :args="{ id: `list${rowIndex}` }"
-          />
-        </div>
-      </div>
-    </div>
-    <div class="col">
-      <button
-        @click="this.attributeDefinitions.splice(rowIndex, 1)"
-        type="button"
-        class="btn btn-sm btn-outline-secondary"
-      >
-        Remove
-      </button>
-    </div>
-  </form>
-  <form class="mt-3">
-    <button
-      @click="
-        this.attributeDefinitions.push({
-          name: '',
-          type: 'STR',
-          key: false,
-          unique: false,
-          required: false,
-          list: false,
-          description: null,
-          bind_to_schema: -1,
-        })
-      "
-      type="button"
-      class="btn btn-sm btn-primary"
-    >
-      Add attribute
-    </button>
-  </form>
-
-  <button @click="sendData" type="button" class="mt-3 btn btn-outline-primary">
-    Create schema
-  </button>
+  <BaseLayout>
+    <template v-slot:additional_breadcrumbs>
+      <li class="breadcrumb-item active">Create new schema</li>
+    </template>
+  </BaseLayout>
+
+  <div class="container">
+    <SchemaEdit :schema="null" ref="editor"></SchemaEdit>
```

Comment:
Is this ref used somehow?

Reply:
No, after defining it I noticed that I don't need it :joy: